### PR TITLE
Select-Screensharing feature

### PR DESCRIPTION
**Pull request by Ivansss**
- [x] Select the screensharing you want to watch.

### CODE REVIEW

**Comment by jancborchardt** [comment]
:+1: as discussed. :)

**Comment by Ivansss** [comment]
@jancborchardt Which icon should we use for the 'Change shared screen' button?

**Comment by jancborchardt** [comment]
cc @nextcloud/spreed and @nextcloud/javascript for review :)

**Comment by nickvergessen** [comment]
@Ivansss is this ready to merge?

**Comment by Ivansss** [comment]
@nickvergessen It is, it just needs a review :)

**Comment by nickvergessen** [comment]
Works nicely, however I found it a bit hard to discover the little icon. maybe we could also have a "…" menu or ">" arrow next to "...´s screen"

But let's do that in a seperate PR

File: js/webrtc.js [hunk 1]
```diff
@@ -576,12 +708,12 @@ var spreedMappingTable = [];
 				userIndicator.className = 'nameIndicator';
 				if (peer) {
 					if (peer.nick) {
-						userIndicator.textContent = peer.nick;
+						userIndicator.textContent = t('spreed', "{participantName}'s screen", {participantName: peer.nick});
```
Comment (nickvergessen) [inline js/webrtc.js]: This does not work for all languages, we need to look into this at some point.
Even in english: `foobas's screen` is incorrect...

File: templates/index-public.php [hunk 2]
```diff
@@ -67,7 +67,23 @@
 				<div class="nameIndicator">
 					<button id="mute" class="icon-audio-white" data-placement="top" data-toggle="tooltip" data-original-title="<?php p($l->t('Mute audio')) ?>"></button>
 					<button id="hideVideo" class="icon-video-white" data-placement="top" data-toggle="tooltip" data-original-title="<?php p($l->t('Disable video')) ?>"></button>
-					<button id="toggleScreensharing" class="icon-screen-off-white screensharing-disabled" data-placement="top" data-toggle="tooltip" data-original-title="<?php p($l->t('Share screen')) ?>"></button>
+					<button id="screensharing-button" class="app-navigation-entry-utils-menu-button icon-screen-off-white screensharing-disabled" data-placement="top" data-toggle="tooltip" data-original-title="<?php p($l->t('Share screen')) ?>"></button>
+		            <div id="screensharing-menu" class="app-navigation-entry-menu">
```
Comment (nickvergessen) [inline templates/index-public.php]: invalid intendation

**Pull request merged**